collect_app/src/test/java/org/odk/collect/android/formlists/blankformlist/BlankFormFixtures.kt

```diff
@@ -0,0 +1,52 @@
+package org.odk.collect.android.formlists.blankformlist
+
+import org.odk.collect.forms.Form
+import org.odk.collect.formstest.FormUtils
+
+object BlankFormFixtures {
```

Comment:
This is definitely nicer, but for me, it's still hard to immediately see in each test what the purpose of each fixture is. For instance, `blankForm4` is important because it's deleted, but I can't tell that without clicking through and then reading the definition carefully. I think we could make it a little easier to understand how a fixture is being set up without needing to click through by using a single function with optional args here:

```kotlin
fun form(formId: String = "1", date: Long = 0, deleted: Boolean = false) {
    return Form.Builder()
        .dbId(1)
        .formId(formId)
        .version("1")
        .displayName("Form ${formId}")
        .date(date)
        .deleted(deleted)
        .formFilePath(FormUtils.createXFormFile(formId, "1").absolutePath)
        .build()
}
```

Then in tests it would be:

```kotlin
@Test
fun `deleted forms should be ignored`() {
    saveForms(
        BlankFormFixtures.form(formId = "1"), 
        BlankFormFixtures.form(formId = "2", deleted = true)
    )
    createViewModel()
        
    assertThat(viewModel.formsToDisplay.value!!.size, `is`(1))
    assertFormItem(viewModel.formsToDisplay.value!![0], BlankFormFixtures.blankForm1)
}
```

I feel like that makes it easier to understand the state that the test is setting up. What do you think?

Reply:
Done.